cli/azd/cmd/testdata/TestUsage-azd-pipeline-config.snap

```diff
@@ -7,7 +7,7 @@ Flags:
   -h, --help                    Gets help for config.
       --principal-name string   The name of the service principal to use to grant access to Azure resources as part of the pipeline.
       --principal-role string   The role to assign to the service principal. (default "Contributor")
-      --provider string         The pipeline provider to use (GitHub and Azdo supported).
+      --provider string         The pipeline provider to use (Github for Github Actions and AzDo for Azure Pipelines). (default "GitHub")
```

Comment:
```suggestion
      --provider string         The pipeline provider to use (github for GitHub Actions and azdo for Azure Pipelines). (default: github)
```

Reply:
IMO, "and" is fine here but you can also use ";". I don't think casing matters. All lower case is easier to type (also see https://learn.microsoft.com/en-us/azure/developer/azure-developer-cli/azd-schema.) So when it's a sample value, all lower case.